network-api/networkapi/sitemaps.py

```diff
@@ -2,6 +2,8 @@
 # https://github.com/wagtail/wagtail/issues/6583#issuecomment-798960446
 from django.contrib.sitemaps import views as sitemap_views
 from wagtail.contrib.sitemaps.sitemap_generator import Sitemap
+from django.shortcuts import render
+from django.conf import settings
```

Comment:
**Suggestion** I know we don't have any checks for it in place, but I find it usually best to keep imports sorted alphabetically.